fixbackend/inventory/inventory_service.py

```diff
@@ -728,3 +745,114 @@ def logs(
+                    "aggregate /change: sum(1) as count | dump"
+                )
+                async with self.client.execute_single(dba, cmd) as result:
+                    changes = {value_in_path(r, "group.change"): value_in_path_get(r, "count", 0) async for r in result}
+                    return (
+                        changes.get("node_created", 0),
+                        changes.get("node_updated", 0),
+                        changes.get("node_deleted", 0),
+                    )
+
+            async def overall_score() -> Tuple[int, int]:
+                current, diff = await progress("account_score", 100, group=set(), aggregation="avg")
+                return current, diff
+
+            (
+                scatters,
+                score_progress,
+                resource_changes,
+                instances_progress,
+                cores_progress,
+                memory_progress,
+                volumes_progress,
+                volume_bytes_progress,
+                databases_progress,
+                databases_bytes_progress,
+                buckets_objects_progress,
+                buckets_size_bytes_progress,
+            ) = await asyncio.gather(
+                resources_per_account_timeline(),
+                overall_score(),
+                nr_of_changes(),
+                progress("instances_total", 0, group=set(), aggregation="sum"),
+                progress("cores_total", 0, group=set(), aggregation="sum"),
+                progress("memory_bytes", 0, group=set(), aggregation="sum"),
+                progress("volumes_total", 0, group=set(), aggregation="sum"),
+                progress("volume_bytes", 0, group=set(), aggregation="sum"),
+                progress("databases_total", 0, group=set(), aggregation="sum"),
+                progress("databases_bytes", 0, group=set(), aggregation="sum"),
+                progress("buckets_objects_total", 0, group=set(), aggregation="sum"),
+                progress("buckets_size_bytes", 0, group=set(), aggregation="sum"),
+            )
+
+            return InventoryInfo(
+                scatters,  # type: ignore
+                score_progress,  # type: ignore
+                resource_changes,  # type: ignore
+                instances_progress,  # type: ignore
+                cores_progress,  # type: ignore
+                memory_progress,  # type: ignore
+                volumes_progress,  # type: ignore
+                volume_bytes_progress,  # type: ignore
+                databases_progress,  # type: ignore
+                databases_bytes_progress,  # type: ignore
+                buckets_objects_progress,  # type: ignore
+                buckets_size_bytes_progress,  # type: ignore
+            )
+
+        return await self.cache.call(
+            compute_inventory_info,
+            key=str(dba.workspace_id) + str(now.replace(minute=0, second=0, microsecond=0)) + str(duration.seconds),
```

Comment:
You should use the workspace_id as key and then pass parameters.
For caching purposes, I would remove now.